test/steps/data.go

```diff
@@ -48,7 +48,7 @@ func (data *Data) RegisterAllSteps(ctx *godog.ScenarioContext) {
 	registerKogitoDataIndexServiceSteps(ctx, data)
 	registerKogitoExplainabilityServiceSteps(ctx, data)
 	registerKogitoTrustyServiceSteps(ctx, data)
-	registerKogitoInfraSteps(ctx, data)
+	//registerKogitoInfraSteps(ctx, data)
```

Comment:
remove?

Reply:
I keep them commented because BDD tests needs to be updated for changes in KOGITO-3039

Reply:
done